dynamic-plugins/wrappers/backstage-plugin-catalog-backend-module-github-dynamic/dist-dynamic/package.json

```diff
@@ -26,7 +26,7 @@
     "test": "backstage-cli package test --passWithNoTests --coverage",
     "clean": "backstage-cli package clean",
     "clean-dynamic-sources": "yarn clean && rm -Rf node_modules",
-    "export-dynamic": "janus-cli package export-dynamic-plugin --embed-package @backstage/plugin-catalog-backend-module-github"
+    "export-dynamic": "janus-cli package export-dynamic-plugin --embed-package @backstage/plugin-catalog-backend-module-github --override-interop default"
```

Comment:
I take it this was missing from some of the dynamic plugins, but should be there on all of them?

Reply:
It's on the wrappers.
In fact this was already set in the main package. The fact that it is added in the `dist-dynamic/package.json` is just the consequence that this chane had not been committed back the last time.

Reply:
@davidfestal